themes/navy/layout/partial/theme.njk

```diff
@@ -5,10 +5,15 @@
     </noscript>
     <img data-src="{{ url_for('themes/screenshots/' + plugin.name + '.png') }}" data-sizes="auto" class="plugin-screenshot-img lazyload" alt="{{ plugin.name }}">
     {% if plugin.preview %}
-    <a href="{{ plugin.preview }}" class="plugin-preview-link" target="_blank"><i class="fa fa-eye"></i></a>
+      <a href="{{ url_for('themes/screenshots/' + plugin.name + '.png') }}" class="plugin-preview-image" target="_blank"><i class="fa fa-eye"></i></a>
     {% endif %}
   </div>
-  <a href="{{ plugin.link }}" class="plugin-name" target="_blank">{{ plugin.name }}</a>
+  <a href="{{ plugin.link }}" class="plugin-name" target="_blank">{{ plugin.name }} <i class="fa fa-external-link"></i></a>
```

Comment:
add `<i class="fa fa-external-link"></i>` to mark it as an external link explicitly.